lib/helpers.coffee

```diff
@@ -159,7 +159,7 @@ module.exports = Helpers =
     unless names instanceof Array
       names = [names]
     startDir = startDir.split(path.sep)
-    while startDir.length && startDir.join(path.sep)
+    while startDir.length and startDir.join(path.sep)
```

Comment:
Looks like this broke the tests: https://travis-ci.org/AtomLinter/atom-linter/builds/91920981